R/load_forecasts.R

```diff
@@ -6,19 +6,21 @@
 #' @param forecast_dates The forecast date of forecasts to retrieve.
 #' Default to all valid forecast dates.
 #' The function will throw an error if all dates in this parameter are invalid forecast dates in Zoltar.
-#' @param locations list of fips. Default to all locations with available forecasts.
+#' @param locations list of location codes Default to all locations with available forecasts.
 #' @param types Character vector specifying type of forecasts to load: “quantile” 
 #' or “point”. Default to all valid forecast types.
 #' @param targets character vector of targets to retrieve, for example
 #' c('1 wk ahead cum death', '2 wk ahead cum death'). 
 #' Default to NULL which stands for all valid targets.
 #' @param source string specifying where forecasts will be loaded from: either 
 #' "local_hub_repo" or "zoltar". Default to "zoltar"
-#' @param hub_repo_path path to local clone of the reichlab/covid19-forecast-hub
+#' @param hub_repo_path path to local clone of the forecast hub
 #' repository
 #' @param as_of a date in YYYY-MM-DD format to load forecasts submitted as of this date. 
-#' Default to NULL to load the latest version. Only available when source is "zoltar" now. 
-#' @param verbose a boolean for printing messages on zoltar job status. Default to TRUE.
+#' Default to NULL to load the latest version.
+#' @param verbose a boolean for printing messages on zoltar job status poll
+#' @param hub character vector, where the first element indicates the hub
```

Comment:
standardizing order of arguments
```suggestion
#' @param hub character vector, where the first element indicates the hub
#' @param verbose a boolean for printing messages on zoltar job status poll
```